app/routes/index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const formBuilder = require('../tools/formTool');
+
+const router = express.Router();
+
+const input1 = {
+  field: 'input',
+  type: 'text',
+  id: 'firstName',
+  label: 'First Name',
+};
+
+const input2 = {
+  field: 'input',
+  type: 'text',
+  id: 'lastName',
+  label: 'Last Name',
+};
+
+const select = {
+  field: 'select',
+  id: 'genderSelect',
+  label: 'Gender',
+  options: ['male', 'female'],
+};
+
+const button = {
+  field: 'button',
+  type: 'submit',
+  id: 'submitBtn',
+  label: 'Submit',
+};
+
+
+/* GET home page. */
+router.get('/', (req, res) => {
```

Comment:
You are going beyond the minimum assignment requirements (and do not specify this with your submission) by including an express server to build and render the form being created, however, you stop short of meaningfully adding  functionality. It would be nice if you built an interface to allow inputs and display the outputs, both rendered and raw text that would allow for full use of the tool you have developed.